bodhi/server/util.py

```diff
@@ -999,26 +999,28 @@ def cmd(cmd, cwd=None):
             basestring.split() to form the list to pass to Popen().
         cwd (basestring or None): The current working directory to use when launching the
             subprocess.
+        exception (bool): If True, raise a RuntimeError if the command's exit code is non-0.
+            Defaults to False.
     Returns:
         tuple: A 3-tuple of the standard output (basestring), standard error (basestring), and the
             process's return code (int).
+    Raises:
+        RuntimeError: If exception is True and the command's exit code is non-0.
     """
-    log.info('Running %r', cmd)
     if isinstance(cmd, six.string_types):
         cmd = cmd.split()
```

Comment:
Most certainly. However, this silly code predates this patch so I would argue we should deal with it separately of this PR. It would not be simple to have this function handle the string case properly. Perhaps it would be better and cleaner to just drop support for str cmds and force all callers to pass lists.

Reply:
Ha! And it appears that the ```cmd``` function wasn't used by anything at all until this PR:

```
$ git grep "cmd(.*)"
bodhi/server/util.py:def cmd(cmd, cwd=None):
bodhi/tests/server/test_util.py:            util.cmd('false')
bodhi/tests/server/test_util.py:        util.cmd('/bin/echo', '"home/imgs/catpix"')
bodhi/tests/server/test_util.py:        util.cmd('/bin/echo', '"home/imgs/catpix"')
bodhi/tests/server/test_util.py:        util.cmd('/bin/echo', '"home/imgs/catpix"')
```

So silly. I think maybe it was used in the old masher code and we forgot to drop it. I should do an analysis sometime of Bodhi's functions to make sure they are all used. I do usually do this when writing test coverage, because it's pointless to test coverage on code that is unused.

In any case, I'll send a PR that just deletes support for a str cmd and I think that'll solve the problem. Nothing uses that feature anyway.

Reply:
I created https://github.com/fedora-infra/bodhi/pull/2332 with this change.